cmd/tools/commentMonitor/main.go

```diff
@@ -132,6 +130,10 @@ func writeArgs(args []string, output string) {
 		if err != nil {
 			log.Fatalln(err)
 		}
+		err = os.Setenv(filename, string(data))
```

Comment:
why is this needed?

Reply:
Since we want to use env vars in the template and `args` such as PR number and branch are not env vars, I had to export those arguments so that I can use them in the comment.

Reply:
ack. Could you add a comment like
```
// Also add all comment args as env variables so that these can be used in the comment template.
```

Reply:
Added the comment.